Libraries/LibGUI/AbstractView.cpp

```diff
@@ -691,8 +691,10 @@ void AbstractView::focusin_event(FocusEvent& event)
 {
     ScrollableWidget::focusin_event(event);
 
-    if (model() && !cursor_index().is_valid())
-        move_cursor(CursorMovement::Home, SelectionUpdate::None);
+    if (model() && !cursor_index().is_valid()) {
+        move_cursor(CursorMovement::Home, SelectionUpdate::ClearIfNotSelected);
```

Comment:
Why is it necessary to change the `SelectionUpdate` mode here? I would think that `SelectionUpdate::None` works fine, as long as we `clear_selection()` after, no?

Reply:
Oh right, that was left in by accident, I’ll change it